packages/components/src/components/SquiggleViewer/index.tsx

```diff
@@ -112,14 +115,16 @@ const component = forwardRef<SquiggleViewerHandle, SquiggleViewerProps>(
     ref
   ) {
     return (
-      <ViewerProvider
-        partialPlaygroundSettings={partialPlaygroundSettings}
-        editor={editor}
-        ref={ref}
-        rootValue={value}
-      >
-        <SquiggleViewerWithoutProvider value={value} />
-      </ViewerProvider>
+      <ErrorBoundary>
```

Comment:
If some users call <SquiggleViewer> directly, it seemed good to have this here, instead of ``ViewerBody``